examples/getting_started/3_Interactivity.ipynb

```diff
@@ -289,12 +289,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Choose the interactive mode for Jupyter Notebook or Lab if desired\n",
-    "#%matplotlib notebook\n",
+    "# Choose the interactive 'widget' mode for Jupyter Notebook or Lab if desired (instead of inline used on the website)\n",
     "#%matplotlib widget\n",
     "%matplotlib inline"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that if you wish to try the interactivity offered by `%matplotlib widget` you will need to pip or conda install the `ipympl` package."
```

Comment:
@jlstevens , is that sufficient? Do they need to reload the web page, relaunch jupyter, or do any other special magic if they've run it once with `inline` and then install ipympl and want to get interactivity?

Reply:
I suppose I should give instructions to restart the kernel and re-run the notebook with `%matplotlib widget` instead of `%matplotlib inline`. Other than that, installing `ipympl` just worked for me (in classic notebook). I did have to restart the server but I don't think my habit of installing packages under the server as it is running is that common...

Reply:
Yes, please do!

Reply:
I see that you merged already so I made the addition in https://github.com/holoviz/datashader/pull/1011